File: sentimental-cash/cash.py
from cs50 import get_float


def main():
    total_coins = 0
    while True:
        try:
            change = get_float("Change owed:")
            if (change) > 0:
                cents = int(change * 100)

                quarters = cents // 25
                cents = cents % 25
                total_coins += quarters

                dimes = cents // 10
                cents = cents % 10
                total_coins += dimes

                nickels = cents // 5
                cents = cents % 5
                total_coins += nickels

                pennies = cents // 1
                total_coins += pennies

                coins = total_coins

                print(coins)

                break
        except ValueError:
            continue


main()

File: cash/cash.c
#include <cs50.h>
#include <stdio.h>
#include <math.h>

int get_cents(void);
int calculate_quarters(int cents);
int calculate_dimes(int cents);
int calculate_nickels(int cents);
int calculate_pennies(int cents);

int main(void)
{
    // Ask how many cents the customer is owed
    int cents = get_cents();

    do
    {
     int x = get_int ("Number of cents:");
    }
    while ( x <= 0);


    // Calculate the number of quarters to give the customer
    int quarters = calculate_quarters(cents);
    cents = cents - quarters * 25;

    // Calculate the number of dimes to give the customer
    int dimes = calculate_dimes(cents);
    cents = cents - dimes * 10;

    // Calculate the number of nickels to give the customer
    int nickels = calculate_nickels(cents);
    cents = cents - nickels * 5;

    // Calculate the number of pennies to give the customer
    int pennies = calculate_pennies(cents);
    cents = cents - pennies * 1;

    // Sum coins
    int coins = quarters + dimes + nickels + pennies;

    // Print total number of coins to give the customer
    printf("%i\n", coins);
}

int get_cents(void)
{
    // TODO
    printf("Change owed: %i \n", cents);
    return 0;
}

int calculate_quarters(int cents)
{
    // TODO

 int calculate_quarters(int cents)
    for (int i = 0; i < 50; i++ )
     {
       return 0;
      }
     else
    {
    return 1;
   }


/////////////////////////////////////////////////////////////////////////

    return 0;
}

int calculate_dimes(int cents)
{
    // TODO
    return 0;
}

int calculate_nickels(int cents)
{
    // TODO
    return 0;
}

int calculate_pennies(int cents)
{
    // TODO
    return 0;
}
